### PR TITLE
Document the downstream SDK repo

### PR DESCRIPTION
**Pull request by jmrodri**
**Description of the change:**
Document the downstream SDK repo. Update the `UPSTREAM-MERGE.sh` script to put the upstream `README.md` into `README-sdk.md`. 

**Motivation for the change:**
Document the downstream repo.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: README-sdk.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<img src="website/static/operator_logo_sdk_color.svg" height="125px"></img>
+
+[![Build Status](https://travis-ci.com/operator-framework/operator-sdk.svg?branch=master)](https://travis-ci.com/operator-framework/operator-sdk)
```
Comment (camilamacedo86) [inline README-sdk.md:3]: should it use the build status of upstream or downstream?

File: README-sdk.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+[![License](http://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
+[![Go Report Card](https://goreportcard.com/badge/github.com/operator-framework/operator-sdk)](https://goreportcard.com/report/github.com/operator-framework/operator-sdk)
```
Comment (camilamacedo86) [inline README-sdk.md:5]: Should not it be pointing out downstream as well?

File: README-sdk.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Documentation
+
+Docs can be found on the [Operator SDK website][sdk-docs].
```
Comment (camilamacedo86) [inline README-sdk.md:9]: Should we not add a link here to open in the upstream/tag which is bumped in downstream?

**Comment by jmrodri** [comment]
@camilamacedo86 and who ever else is reviewing. What you are seeing in this PR are 2 things.

File: README-sdk.md (hunk 1 above)
Comment (jmrodri) [reply README-sdk.md:3]: upstream, see my new comment https://github.com/openshift/ocp-release-operator-sdk/pull/102#issuecomment-779416485

File: README-sdk.md (hunk 2 above)
Comment (jmrodri) [reply README-sdk.md:5]: No, README-sdk-md is strictly a copy of the UPSTREAM. Left untouched, just moved it out of the way.  See https://github.com/openshift/ocp-release-operator-sdk/pull/102#issuecomment-779416485

File: README-sdk.md (hunk 3 above)
Comment (jmrodri) [reply README-sdk.md:9]: Nope, README-sdk.md is strictly a copy of upstream README.md https://github.com/openshift/ocp-release-operator-sdk/pull/102#issuecomment-779416485

**Comment by jmrodri** [comment]
@camilamacedo86 I redid the commits so that the diff is more obvious of what I'm actually changing.

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

**Pull request merged**